src/port1.0/portconfigure.tcl

```diff
@@ -511,8 +511,17 @@ proc portconfigure::configure_get_universal_ldflags {} {
 }
 
 # internal proc to determine if the compiler supports -arch
-proc portconfigure::arch_flag_supported {compiler} {
-    return [regexp {^gcc-4|llvm|apple|clang} $compiler]
+proc portconfigure::arch_flag_supported {compiler {multiple_arch_flags no}} {
+    if {${multiple_arch_flags}} {
+        return [regexp {^gcc-4|llvm|apple|clang} ${compiler}]
+    } else {
+        # GCC prior to 4.7 does not accept -arch flag
```

Comment:
This, combined with the previous commit, seem problematic.  Just because the compiler supports -arch does not imply that it supports taking *multiple* -arch options on the command line.  gcc-4.7 can take -arch, but only *one*.  It cannot be used to make universal binaries.

Reply:
Actually, I think I misread that.  This looks fine.